templates/presenter.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+/**
+ * Formats data for the `` page
```

Comment:
We could maybe generate this from the folder paths. But it looks a but complicated. 

We could put a placeholder for the route and then it's a simple find and replace.